textile/features.textile

```diff
@@ -910,6 +934,27 @@ h3(#objects). Objects
 
 Reserved for @Objects@ feature specification. Reserved spec points: @OBJ@, @LM@, @LC@
 
+h3(#realtime-annotations). RealtimeAnnotations
+
+* @(RTAN1)@ @RealtimeAnnotations#publish@ function:
+** @(RTAN1a)@ Must accept the same arguments and performs the same validation, field setting, and data encoding as in @RSAN1@
+** @(RTAN1b)@ Has the same connection and channel state conditions as message publishing, see @RTL6c@
+** @(RTAN1c)@ Must put the annotation to be published into an array in the @annotations@ field of a @ProtocolMessage@ whose @action@ field is set to @ANNOTATION@, and @channel@ field is set to the channel name, and send it to the connection
```

Comment:
In this case we didin't specify as `create` => https://github.com/ably/specification/pull/292/files#r2137030102